packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1775,6 +1776,38 @@ class PubWarningText extends StatelessWidget {
   }
 }
 
+class InternalFlutterWebWarningText extends StatelessWidget {
+  const InternalFlutterWebWarningText({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return RichText(
+      text: TextSpan(
+        children: [
+          TextSpan(
+              text:
+                  'Warning: Flutter DevTools is not currently supported for Flutter Web apps.\n\n',
+              style: theme.subtleTextStyle
+                  .copyWith(color: theme.colorScheme.errorTextColor)),
+          TextSpan(
+              text: 'Some debugging features might not work as expected.\n',
+              style: theme.subtleTextStyle),
+          TextSpan(text: 'See ', style: theme.subtleTextStyle),
+          TextSpan(
```

Comment:
Use LinkTextSpan from common_widgets.dart. This widget uses logic from launch_url.dart, which has conditional import logic so that we don't try to use url_launcher on Desktop, which will result in a crash (or at least it used to)

Reply:
Done, thanks!